setup.cfg

```diff
@@ -51,6 +51,11 @@ install_requires =
     six
     sqlalchemy
     subprocess32
+zip_safe = False
+include_package_data = True
+
+[options.package_data]
+* = *.cfg, *.sql
```

Comment:
For my info, what the previous lines end up doing?

Reply:
They include the cfg and sql files inside the wheel + tarball which gets uploaded to PyPI. This also means they're included in the installed files in `lib/pythonX.Y/site-packages/DIRAC/`.

Reply:
Why including the cfg files?

Reply:
IIRC they're used when setting up the installation: https://github.com/DIRACGrid/DIRAC/blob/e0c0b1a94f3007ecc5e83f4bfd2964844a28defb/src/DIRAC/FrameworkSystem/Client/ComponentInstaller.py#L788-L793

Reply:
Ah, right. But please, only include "ConfigTemplate.cfg" files, not *.cfg.